src/Generator.Bind/EmbeddedResources.cs

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
This isn't consistent with the rest of the project

Reply:
Granted, this is the header suggested by the style guide, but I'd rather keep the license header the same as the old license header until we have oficially confirmed our license and its associated header.

Reply:
Okay, i'll change the header.